### PR TITLE
chore(global): update to version 8.27.0

### PR DESCRIPTION
**Pull request by alex00**
## Description

This PR make a big major update from **5.3.0** to **8.27.0** of eslint

For this update we need to put some rules on `["warn"]`

### Eslint rules
- [`arrow-body-style`](https://eslint.org/docs/latest/rules/arrow-body-style)
- [`prefer-regex-literals`](https://eslint.org/docs/latest/rules/prefer-regex-literals)
- [`no-promise-executor-return`](https://eslint.org/docs/latest/rules/no-promise-executor-return)
- [`prefer-object-spread`](https://eslint.org/docs/latest/rules/prefer-object-spread) (for this one we do not put off on eslint config and only add a comment tp deactivate it on ui-jobteaser https://github.com/jobteaser/ui-jobteaser/pull/2059/files#diff-e58bca043a45aee3730d33a28613b9511ac5ca05801346f44a96944208e1d9d0R29)
- [`default-param-last`](https://eslint.org/docs/latest/rules/default-param-last)

### Import eslint rules
- [`import/no-import-module-exports`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-import-module-exports.md)

### React eslint rules
- [`react/no-unused-class-component-methods`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unused-class-component-methods.md)
- [`react/function-component-definition`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/function-component-definition.md)
- [`react/jsx-props-no-spreading`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md)
- [`react/jsx-no-constructed-context-values`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-constructed-context-values.md)
- [`react/jsx-curly-brace-presence`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md)
- [`react/static-property-placement`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/static-property-placement.md) -> only 1 present in https://github.com/jobteaser/ui-jobteaser/blob/master/web_app/src/shared_spa/components/elements/PageViewTrackerV3/index.js#L14

## Trooblshooting

For the Visual Studio Code users if you have this error `Eslint dosent work on VScode Cliengine is not supported anymore`
-> you need to use a recent version like **1.73.1**, [you can download it here](https://code.visualstudio.com/)

## Jira Ticket

https://jobteaser.atlassian.net/browse/OP-2062

## Related PR

- https://github.com/jobteaser/ui-jobteaser/pull/2059
- https://github.com/jobteaser/ui-kit/pull/652

### CODE REVIEW

**Comment by Zeboch** [comment]
As discussed in Chapter Life can you please change the rule to "warning" instead of "off" and add a max warning on the npm script plz ?

So the linter scope is preserved and we are still aware about the issues (remember last time we did it we start around 2k warnings).

Then we could discuss about removing some unnecessary rules with Chapter!

Thanks 🙏

**Review by vasconita — APPROVED** [review]
Thanks a lot!

File: rules/base.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = {
     node: true
   },
   rules: {
-    "arrow-body-style": ["error", "as-needed"],
+    "arrow-body-style": ["warn"],
```
Comment (Zeboch) [inline rules/base.js:9]: Be careful that you removed the option of the rule here.

**Review by Zeboch — APPROVED** [review]
Just a quick comment about a modification you did.

**Pull request merged**